pkg/scheduler/framework/plugins/nodeports/node_ports_test.go

```diff
@@ -212,48 +220,15 @@ func TestGetContainerPorts(t *testing.T) {
 					Protocol:      v1.ProtocolTCP,
 				},
 			}).Obj(),
-			expected: []*v1.ContainerPort{
```

Comment:
You should add the `HostPort` to all the cases above, and in addition, add a new container with no `HostPort` set.

Reply:
+1